joomla/joomla-cms
# To many elements to index

I don't know why in the query `#__tags` is twice. Once as a `a` and second time as a `b`. For me it works perfectly when we remove `b`. I had a problem with this - for 1100 tags on my websites, Smart Search Indexer found more than 1233000 items to index.

Pull Request for Issue #13866.

### Summary of Changes

Remove `#__tags AS b` from the query and change the `b.created_user_id` to the `a.created_user_id` in the JOIN part of the query.

### Testing Instructions

Use `php cli/finder_indexer.php` to check if the number of tags is the same as a number of items to index.

### Expected result

For the number of tags have the same number of items for index.

### Actual result

For 1100 tags, I have more than 1233000 items to index.

### Documentation Changes Required
